blocks/bricks/sequence_generators.py

```diff
@@ -440,7 +440,15 @@ class SoftmaxEmitter(AbstractEmitter, Initializable, Random):
 
     Interprets readout elements as energies corresponding to their indices.
 
+    Parameters
+    ----------
+    initial_output : int or a scalar :class:`~theano.Variable`
+        The initial output.
```

Comment:
This is not very easy to understand without reading the code. I would say something like "The label (token) to output for the first time step of the :class:`.SequenceGenerator`. This corresponds to the 'beginning-of-sentence' token."